test/rooms_test.js

```diff
@@ -445,6 +446,48 @@ describe("/rooms", function() {
         .expect(400)
         .end(done);
       });
+
+      it("should fail if both context and roomName are present", function(done) {
```

Comment:
In this case, should we actually ignore the roomName and use just the context rather than failing?